release/7zip.xml

```diff
@@ -12,35 +12,53 @@
 
     This program is distributed in the hope that it will be useful,
     but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
     GNU General Public License for more details.
 
     You should have received a copy of the GNU General Public License
-    along with this program. If not, see <http://www.gnu.org/licenses/>.
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+    @app 7-Zip
     @url https://www.7-zip.org/
-    @cleanerversion v???
-    @cleanerdate 2019-03-11
+    @url http://p7zip.sourceforge.net/
+    @os Windows, Linux
```

Comment:
The application is for both, but the cleaner is just for Windows. It seems the Linux 7-Zip does not need a cleaner

Reply:
Plz see my comment above! I think this cleaner should (later) clean the 7-Zip Linux GUI, too! My work - so far - are just preparations for that (in comments).